docs/src/main/tut/plots.md

```diff
@@ -73,79 +73,103 @@ EvilPlot takes an unusual approach to overlaying, in the spirit of building more
 ones. When you want to make a multilayered plot, just make a plot for each layer. We then give you a combinator,
 `Overlay`, to compose all the plots into a meaningful composite.
 
-Let's look at two equally meaningful plots. The first is a
-[Ramachandran plot](https://en.wikipedia.org/wiki/Ramachandran_plot), which is a plot of the phi and psi dihredral
-angles in a peptide bond. This data came from a molecular dynamics simulation of alanine dipeptide, so we have a
-computed "phi" and "psi" from the simulation for each iteration. We want to look at how these two angles were
-distributed, so let's make a contour plot out of the density.
+Let's take a look at some real world data. In its natural state, real world data can be hard to interpret and even harder
+to put to use. Take a look at the public data from https://data.boston.gov/dataset/wicked-free-wi-fi-locations
```

Comment:
- This URL isn't hyperlinked.
- You just said "take a look" two sentences previously.  
- I don't understand why "Latitude(x) and Longitude(y)" are capitalized and look like functions of x and y respectively.
- You mention "parse this file", but you haven't made clear you are talking about the csv file.  (Your URL was a link to a website containing many files.)

Maybe rewrite as:
We will use [Wicked Free Wi-Fi](https://data.boston.gov/dataset/wicked-free-wi-fi-locations), a public data set containing the latitude and longitude (columns X and Y respectively) of several free wi-fi hotspots in Boston.  After downloading the CSV file, let's parse the data into a `Seq[Points]`:

Reply:
Reworded and hyperlinked the URL. Thanks for the rewrite suggestion!